BooruSharp/Search/Post/SearchResult.cs

```diff
@@ -15,8 +15,10 @@ namespace BooruSharp.Search.Post
         /// <param name="fileUrl">The URI of the file.</param>
         /// <param name="previewUrl">The URI of the preview image.</param>
         /// <param name="postUrl">The URI of the post.</param>
+        /// <param name="sampleUri">The URI of the sample image.</param>
```

Comment:
What is the difference between sample and preview?

Reply:
Preview is a thumbnail, sample is a slightly resized version of the full image that's intended to load faster.

Reply:
Do you think that maybe you could explain that in the comment? Would be easier to understand for people that are not aware about it